.pax/prepare-workspace.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash -e
+
+################################################################################
+# This program and the accompanying materials are made available under the terms of the
+# Eclipse Public License v2.0 which accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Copyright IBM Corporation 2018, 2020
+################################################################################
+
+################################################################################
+# Build script
+# 
+# - build client
+################################################################################
+
+# contants
+SCRIPT_NAME=$(basename "$0")
+BASEDIR=$(dirname "$0")
+PAX_WORKSPACE_DIR=.pax
+PACKAGE_NAME=$(node -e "console.log(require('./package.json').name)")
+PACKAGE_VERSION=$(node -e "console.log(require('./package.json').version)")
+PACKAGE_DESC=$(node -e "console.log(require('./package.json').description)")
+
+cd $BASEDIR
+cd ..
+ROOT_DIR=$(pwd)
+
+# prepare pax workspace
+echo "[${SCRIPT_NAME}] cleaning PAX workspace ..."
+rm -fr "${PAX_WORKSPACE_DIR}/content"
+mkdir -p "${PAX_WORKSPACE_DIR}/content"
+
+# build client
+echo "[${SCRIPT_NAME}] building client ..."
+cd "dist"
+npm install
```

Comment:
If we re-run `npm install` on z/OS side, we can skip this step. And this can avoid transferring node_modules folder to z/OS.

Reply:
moved to pre-package.sh